index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
+    </header>
+    <nav>
+      <a href="#" target="_parent" class="logo">My Portfolio</a>
+      <img src="./assets/Union.png" class="mobile_menu_icon"/>
+    </nav>
+
+    <section>
+      <div class="hero">
+        <p class="author">I’m Tanveer Glad to see you!</p>
+        <p class="bio">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+           don’t hestiate to contact me.</p>        
+           <a href="#" class="connect_button">let's connect</a>
+      </div>      
+    </section>
+    <aside>
+      <ul>
+        <li><img src="./assets/Vector.svg" alt="twitter"></li>
+        <li><img src="./assets/Linkedin icon.svg" alt="Linkedin"></li>
+        <li><img src="./assets/Vector-2.svg" alt="m-icon"></li>
+        <li><img src="./assets/Vector-1.svg" alt="github"></li>
+        <li><img src="./assets/Vector-3.svg" alt="victory"></li>
```

Comment:
- Let's wrap each image in an anchor tag as those should link to your social media platforms.